src/telemetry/telemetry.c

```diff
@@ -72,15 +71,12 @@ telemetry_parse_version(const char *json, VersionInfo *installed_version, Versio
 	 * Now parse the version string. We expect format to be
 	 * XX.XX.XX-<prerelease_tag>, and if not, we error out
 	 */
-	if (!version_parse(result->versionstr, &result->vinfo))
+	if (!version_parse(result->versionstr))
```

Comment:
why do we even need this check? Previously we needed this to parse the version to compare them. But why do we need to check the format now? Maybe we want to just check strlength and for printable characters?

Reply:
I don't think sql injection is an issue here?

Reply:
Yeah, I didn't feel strongly about this. I was happy just printing the first 6 characters of the string that's returned from the server. But others suggested we parse to make sure it's "well-formed".

Reply:
FWIW, I am leaning towards removing since doing the version check on the server means we don't need to know the format anymore.

Reply:
What I ended up doing is print a fixed number of chars (6, but we can always change that via the macro), and making sure that we only print either alphanumeric or '.' chars.